noir/compiler/noirc_evaluator/src/ssa/ir/instruction/call.rs

```diff
@@ -417,7 +417,7 @@ fn simplify_black_box_func(
                 _ => SimplifyResult::None,
             }
         }
-
+        BlackBoxFunc::Poseidon2Permutation => SimplifyResult::None, //TODO(Guillaume)
```

Comment:
can you create an issue in Noir for this?

Reply:
https://github.com/noir-lang/noir/issues/4170